MapboxCoreNavigation/Route.swift

```diff
@@ -0,0 +1,13 @@
+import MapboxDirections
+
+extension Route {
+    
+    var json: String? {
```

Comment:
Encoding as JSON is expensive enough that we probably shouldn’t hand-wave away the fact that this computed variable creates a new structure every time it is accessed. Instead, make it a method.

Once we implement mapbox/MapboxDirections.swift#391, the original JSON data can live alongside the decoded Route in a Response object.